conf/configuration.go

```diff
@@ -20,6 +20,7 @@ import (
 const (
 	defaultAccountsHost = "10.10.10.70"
 	defaultAccountsPort = "3000"
+	defaultLogFile      = "/logs/pinner.log"
```

Comment:
Don't you mean `./logs/pinner.log` to make it relative to where the pinner is running from?
If it's absolute, this won't run outside of docker without root privileges

Reply:
On the same note, can you add a test which makes sure that the log file is created at the default path and if overwritten by the environment var at the specified path?

Reply:
The idea is for this to only work in docker, so yes, I mean the file at the root `/log` directory.
If somebody wants to run it outside of docker without root, they can pass the `PINNER_LOG_FILE` env var and set this value to whatever they want.

Reply:
> On the same note, can you add a test which makes sure that the log file is created at the default path and if overwritten by the environment var at the specified path?

Sure. Do you mind if I do that in a F/U, so we don't need to wait for Matt to be back from vacation or involve another reviewer?

Reply:
Here's the F/U: https://github.com/SkynetLabs/pinner/pull/38

Reply:
A f/u is fine if you have manually verified it's working. The goal is simply to verify this MR is doing what it claims to do. 1. log to disk 2. let you configure the log location.

Regarding the log location it's true that we ship using docker but we also want to run services locally for development and run tests locally without docker. Having the default experience be an error because the logfile can't be created is very annoying.

Reply:
Our current situation is "never log to disk". This PR changes that to "allow logging to disk on demand". I don't see a reason to jump all the way to "always log to disk".